src/test/java/com/wikia/webdriver/testcases/Anonymization/TestFile.java

```diff
@@ -0,0 +1,10 @@
+package com.wikia.webdriver.testcases.Anonymization;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.openqa.selenium.remote.RemoteWebDriver'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUselessImportCheck)